### DISCUSSION TITLE
Doorbell setup successful - However chimes multiple times upon triggering

### DISCUSSION THREAD

**Discussion by Stormeh** [potmat/homebridge-google-nest-sdm#84] [Q&A] [answered]
First and foremost - Thank you so much for maintaining this.

I tried this today - I was able to setup everything successful - Only issue I face now is when I ring the doorbell, 3 notifications will come in over time and my homepods will chime 3 times.  I am absolutely baffled by this and don't know where to start with the troubleshooting, any ideas?

**Comment by potmat** [comment, marked as answer]
Also reported here: https://github.com/potmat/homebridge-google-nest-sdm/issues/85

**Reply by Stormeh** [reply]
Yeah sorry about that, I went thru the log afterwards and saw that so raised an issue instead, feel free to close one of them, my bad.
